cpu/esp8266/README.md

```diff
@@ -0,0 +1,266 @@
+# RIOT-OS port for ESP8266 and ESP8285
+
+There are two implementations that can be used:
+
+- the **SDK version** which is realized on top of an SDK (esp-open-sdk or ESP8266_NONOS_SDK and
+- the **non-SDK version** which is realized without the SDK.
+
+The non-SDK version produces a much smaller code size than the SDK version and is more efficient in excecution because it does not need to run additional SDK functions to keep the SDK system alive.
+
+The **non-SDK** version is probably the **best choice if you do not need the built-in WiFi module**, for example, when you plan to connect the MCU to an IEEE 802.15.4 radio module for communication.
+
+By default, the non-SDK version is compiled. To compile the SDK-version, add option ```SDK=1``` to the make command, e.g.,
+
+    make flash BOARD=esp8266-esp-12x -C test/shell SDK=1 ...
+
+For more information, see section **Compile Options**.
+
+## Tool Chain
```

Comment:
Hi @gschorcht 

I think this information should be in the wiki (e.g check [MIPS family](https://github.com/RIOT-OS/RIOT/wiki/Family%3A-MIPS))

Reply:
I agree completely. When I was asking the community whether I should already start to create a wiki entry before the code is merged, I got the answer that there is a switch from wiki to doxygen in progress. So, I thought to leave these necessary information within a REAMDE for the moment.

Could we leave the README there for the moment and remove them later when I have created a wiki entry? Without these information a user will not be able to compile the code.

Reply:
yes sure, makes sense. Let's go that way